fs/README.md

```diff
@@ -0,0 +1,160 @@
+# fs
+
+fs module is made to provide helpers to manipulate the filesystem.
+
+## Usage
+
+All the following modules are exposed in `mod.ts`
+
+### empty_dir
```

Comment:
```suggestion
### emptyDir
```

Reply:
Shouldn't it be the same as the method?

Reply:
My bad, was looking at filename not exposed function name.